src/Process/Pool/Logger/Message.php

```diff
@@ -4,18 +4,22 @@
 
 namespace Neighborhoods\Kojo\Process\Pool\Logger;
 
+use Neighborhoods\Kojo\Data\JobInterface;
+
 class Message implements MessageInterface, \JsonSerializable
 {
     const KEY_TIME = 'time';
     const KEY_LEVEL = 'level';
     const KEY_PROCESS_ID = 'process_id';
     const KEY_PROCESS_PATH = 'process_path';
     const KEY_MESSAGE = 'message';
+    const KEY_JOB_METADATA = 'kojo_job';
 
     protected $time;
     protected $level;
     protected $process_id;
     protected $process_path;
+    protected $kojo_job = [];
```

Comment:
In the `Process\Pool\Logger\Formatter`, there's branching logic for a log format called "pipes". I dunno what that's used for (could be vestigial), but it explicitly reads properties off the message, meaning currently it'll skip over `kojo_job`

Reply:
`pipes` is our original column based output structure for viewing in a terminal:
<img width="1440" alt="Screen Shot 2019-08-08 at 10 43 37 AM" src="https://user-images.githubusercontent.com/1881846/62717542-b0f63000-b9c9-11e9-8692-59d2451a7aa0.png">

I could pull a few items out of the `Data/Job` object like `type_code` and `assigned_state`. Do you think there are any other key items to display @mucha55? Full list of possible keys on the [JobInterface](https://github.com/neighborhoods/kojo/blob/d116e30ca237b6530f60924eca32e79dc42e7c2f/src/Data/JobInterface.php#L8)

Reply:
Haha, I never knew about this

So it's pretty much just for debugging and there's no reason to use it on prod? In that case it's up to you, obviously adding too many things will negate the benefits of this format. I could see removing PID (it's redundant) and adding `type_code` and `assigned_state`, but I don't feel strongly about it